lib/local-links-manager/import/links.rb

```diff
@@ -34,12 +34,9 @@ def import_links(csv_string)
           begin
             link.update!(url: new_url)
             updated += 1
-          rescue ActiveRecord::RecordInvalid => e
-            GovukError.notify(
-              LocalLinksManager::Import::UrlValidationException.new(e.message),
-              extra: slugs.merge(link_id: link.id),
-            )
+          rescue ActiveRecord::RecordInvalid
             error_message = "Line #{index}: invalid URL '#{new_url}'"
+            Rails.logger.warn("#{error_message} (#{slugs.merge(link_id: link.id)})")
```

Comment:
I think we should log the full exception (`e.message`) in case we need to debug something and the URL _looks_ valid so knowing the full exception would be beneficial.

Reply:
Sounds good 👍